modules/fedimint-ln-client/src/lib.rs

```diff
@@ -272,7 +292,7 @@ impl LightningClientExt for Client {
             &mut dbtx,
             operation_id,
             LightningCommonGen::KIND.as_str(),
-            LightningMeta::Receive {
+            LightningMetaVariants::Receive {
```

Comment:
This is the bug I think. `finalize_and_submit_transaction` will actually add an operation log entry so we don't need to do this here. I think if you keep this you should see a warning on the CLI when doing receives

Reply:
Yep, there is a fix in #2486, so there will be a conflict https://github.com/fedimint/fedimint/pull/2486/files#diff-d754f9b1f77285e47d11bdecd4e3df652f846e5cedd12ded095ae84816a64cc2L205-L217